shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/domain/usecase/SurveyListUseCase.kt

```diff
@@ -0,0 +1,23 @@
+package co.nimblehq.blisskmmic.domain.usecase
+
+import co.nimblehq.blisskmmic.domain.model.Survey
+import co.nimblehq.blisskmmic.domain.model.Token
+import co.nimblehq.blisskmmic.domain.repository.SurveyRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+
+interface SurveyListUseCase {
+
+    operator fun invoke(page: Int): Flow<List<Survey>>
+}
+
+class SurveyListUseCaseImpl(
+    private val repository: SurveyRepository
+    ) : SurveyListUseCase {
```

Comment:
Please refer to the AS style settings above for fixing the indentation here as well 🙏
```suggestion
class SurveyListUseCaseImpl(
    private val repository: SurveyRepository
) : SurveyListUseCase {
```

Reply:
I am using the same settings as your image but I need to add `,` so that the line break remove the indent. 🤔

Reply:
hmm, I don't have that issue with my AS's `Option+Command+I` so weird 🤔

https://user-images.githubusercontent.com/70877098/206682934-be3e02bf-f48b-494f-8a19-e998431c8604.mov

I have just exported the XML style for you here, maybe can you import it to your AS and try again 🙏
[Android_Project.xml.zip](https://github.com/blyscuit/kmm-swiftUI-IC/files/10194017/Android_Project.xml.zip)

Reply:
https://user-images.githubusercontent.com/6356137/206687023-49c19c09-0e67-4d71-8ea8-468a8ae2ff96.mov

Same thing still happen, I also tried adding line from AS, and the `,` magically appear.

Reply:
@blyscuit, can you remove the comma and try this option `Option+Command+I` for automatically indenting lines according to the style and see if it works for you?

![Screenshot 2022-12-09 at 18 11 40](https://user-images.githubusercontent.com/70877098/206689882-137bd617-5991-4a8b-9c8e-d93db79f3a7a.png)

Reply:
Just sharing, one thing Android devs usually do before pushing code is to press the combination of `Option+Command+I` and `Option+Command+O` for having a quick round of refactoring the code on the changed files. 🚀

Reply:
@minhnimble This works. Thank you.